composer.json

```diff
@@ -27,7 +27,8 @@
         "symfony/options-resolver": "^2.3 || ^3.0",
         "sonata-project/doctrine-orm-admin-bundle": "^3.0",
         "sonata-project/admin-bundle": "^3.0",
-        "sonata-project/datagrid-bundle": "^2.2"
+        "sonata-project/datagrid-bundle": "^2.2",
+        "cocur/slugify": "^1.4 || ^2.0"
```

Comment:
Passing and merged :) @Vesax , can you please rebase? You should get a conflict, fix it by adding the package at the right place please.